APPS.md

```diff
@@ -10,18 +10,17 @@ Name | Description | Logo
 [Meetup for iPhone](https://itunes.apple.com/app/apple-store/id375990038?pt=288116&ct=apipage&mt=8) |  Now while you're on the go, you can use your iPhone to connect with Meetup! | ![Meetup for iPhone logo](http://img2.meetupstatic.com/img/055891063038236603711/app_download/ios/download.en.svg)
 [Amazing Prize-O-Tron](http://prize-o-tron.herokuapp.com/) | Do you give away prizes at your events? You can use the Amazing Prize-O-Tron to quickly and easily randomly select prize winners from one or more meetup events. | ![Amazing Prize-O-Tron](http://photos1.meetupstatic.com/photos/event/5/9/f/3/highres_12923027.jpeg)
 [Clique](http://bit.ly/clique4wp8) | The unofficial Meetup app for Windows Phone 8, is the premiere app for Meetup and is the first full-function Meetup app in the Windows Phone Store. | ![Clique logo](http://photos4.meetupstatic.com/photos/event/d/9/b/e/highres_312175742.jpeg)
-[CityPulse](http://www.citypulse.io/) | Discover your city with CityPulse - citypulse.io. We're passionate about putting meetups and other events smack bang on the map using a beautiful and unique interface. | ![CityPulse logo](http://photos1.meetupstatic.com/photos/event/4/6/c/3/highres_439338115.jpeg)
-[Daytripper](https://play.google.com/store/apps/details?id=com.daytripper.app) | Daytripper is an Android and Kindle app to find local businesses, ticketed events, and meetups. The app is primarily designed for people who have little proficiency with computers. All input comes from voice and text. To use the app, just type your query in the search box. Or if your device supports voice search, click the microphone icon and speak your query. Please see the help menu for query details with examples. | ![Daytripper logo](http://photos1.meetupstatic.com/photos/event/2/c/9/event_435540713.jpeg)
+[Daytripper](https://play.google.com/store/apps/details?id=com.vocifery.daytripper) | Daytripper is an Android and Kindle app to find local businesses, ticketed events, and meetups. The app is primarily designed for people who have little proficiency with computers. All input comes from voice and text. To use the app, just type your query in the search box. Or if your device supports voice search, click the microphone icon and speak your query. Please see the help menu for query details with examples. | ![Daytripper logo](http://photos1.meetupstatic.com/photos/event/2/c/9/event_435540713.jpeg)
 [Drupal Meetup Events](http://drupal.org/project/meetup_events) | A Drupal module to automatically publish and update events from a drupal site to a meetup group. | ![Drupal Meetup Events logo](http://photos1.meetupstatic.com/photos/event/5/9/f/3/highres_12923027.jpeg)
```

Comment:
I think we can leave out photos for now. Uploading a photo was part of the previous submission process. I think we're going to want to do away with that so ability to include a photo will be more awkward now